sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/TableResourcesOperationsTests.cs

```diff
@@ -0,0 +1,113 @@
+
+            var tableCreateUpdateParameters2 = new TableCreateUpdateParameters(new TableResource(tableName), new CreateUpdateOptions(sampleThroughput, new AutoscaleSettings()));
+            TableGetResults tableGetResults3 = (await WaitForCompletionAsync(await cosmosDBMgmtClient.TableResources.StartCreateUpdateTableAsync(resourceGroupName, databaseAccountName, tableName, tableCreateUpdateParameters2))).Value;
+            Assert.IsNotNull(tableGetResults3);
+            Assert.AreEqual(tableName, tableGetResults3.Resource.Id);
+            actualThroughput = (await cosmosDBMgmtClient.TableResources.GetTableThroughputAsync(resourceGroupName, databaseAccountName, tableName)).Value.Resource.Throughput;
+            Assert.AreEqual(sampleThroughput, actualThroughput);
+
+            TableGetResults tableGetResults4 = (await cosmosDBMgmtClient.TableResources.GetTableAsync(resourceGroupName, databaseAccountName, tableName)).Value;
+            Assert.IsNotNull(tableGetResults4);
+            VerifyTables(tableGetResults3, tableGetResults4);
+
+            List<TableGetResults> tables = await cosmosDBMgmtClient.TableResources.ListTablesAsync(resourceGroupName, databaseAccountName).ToEnumerableAsync();
+            Assert.IsNotNull(tables);
+            Assert.AreEqual(1, tables.Count);
+            VerifyTables(tableGetResults4, tables[0]);
+
+            ThroughputSettingsGetResults throughputSettingsGetResults1 = await WaitForCompletionAsync(await cosmosDBMgmtClient.TableResources.StartMigrateTableToAutoscaleAsync(resourceGroupName, databaseAccountName, tableName));
+            Assert.IsNotNull(throughputSettingsGetResults1.Resource.AutoscaleSettings);
+            Assert.AreEqual(defaultMaxThroughput, throughputSettingsGetResults1.Resource.AutoscaleSettings.MaxThroughput);
+            Assert.AreEqual(defaultThroughput, throughputSettingsGetResults1.Resource.Throughput);
+
+            ThroughputSettingsGetResults throughputSettingsGetResults2 = await WaitForCompletionAsync(await cosmosDBMgmtClient.TableResources.StartMigrateTableToManualThroughputAsync(resourceGroupName, databaseAccountName, tableName));
+            Assert.IsNull(throughputSettingsGetResults2.Resource.AutoscaleSettings);
+            Assert.AreEqual(defaultMaxThroughput, throughputSettingsGetResults2.Resource.Throughput);
+
+            var throughputSettingsUpdateParameters = new ThroughputSettingsUpdateParameters(new ThroughputSettingsResource(defaultThroughput, null, null, null));
+            ThroughputSettingsGetResults throughputSettingsGetResults = (await WaitForCompletionAsync(await cosmosDBMgmtClient.TableResources.StartUpdateTableThroughputAsync(resourceGroupName, databaseAccountName, tableName, throughputSettingsUpdateParameters))).Value;
+            Assert.AreEqual(defaultThroughput, throughputSettingsGetResults.Resource.Throughput);
+        }
+
+        private void VerifyTables(TableGetResults expectedValue, TableGetResults actualValue)
```

Comment:
Move this to CosmosDBTestUtilities

Reply:
It only used in this test, I think no need to move.